src/Resources/skeleton/crud/controller/Controller.tpl.php

```diff
@@ -13,11 +13,18 @@
 use Symfony\Component\Routing\Annotation\Route;
 
 /**
+ * <?= $class_name ?> class
  * @Route("<?= $route_path ?>")
  */
-class <?= $class_name ?> extends <?= $parent_class_name; ?><?= "\n" ?>
+class <?= $class_name ?> extends <?= $parent_class_name; ?><?= PHP_EOL ?>
 {
     /**
+     * <?= ucwords($route_name) ?> index page
```

Comment:
In this method we can remove the generic comment, the `@param` and the `@return` because IDEs can infer all that with the arguments type hints and the return types.

Same in the other methods of this class.